azure-pipelines-product-construction-service.yml

```diff
@@ -154,119 +154,120 @@ stages:
-                $before = az containerapp show --name $(containerappName) -g $(resourceGroupName) --output json
-                Set-Content -Path $(diffFolder)/before.json -Value $before
-            displayName: Snapshot configuration (before)
+      - task: AzureCLI@2
+        inputs:
+          azureSubscription: $(serviceConnectionName)
+          scriptType: pscore
+          scriptLocation: inlineScript
+          inlineScript: |
+            New-Item -ItemType Directory -Path $(diffFolder)
+            $before = az containerapp show --name $(containerappName) -g $(resourceGroupName) --output json
+            Set-Content -Path $(diffFolder)/before.json -Value $before
+        displayName: Snapshot configuration (before)
 
-          - download: current
-            displayName: Download ProductConstructionService.Deployment
-            artifact: ProductConstructionService.Deployment     
+      - download: current
+        displayName: Download ProductConstructionService.Cli
+        artifact: ProductConstructionService.Cli     
 
-          - task: NuGetToolInstaller@1
-            displayName: Use NuGet
-            inputs:
-              versionSpec: 5.3.x
+      - task: NuGetToolInstaller@1
+        displayName: Use NuGet
+        inputs:
+          versionSpec: 5.3.x
 
-          - powershell: |
-              . .\eng\common\tools.ps1
-              InitializeDotNetCli -install:$true
-              .\.dotnet\dotnet workload install aspire
-            displayName: Install .NET and Aspire Workload
+      - powershell: |
+          . .\eng\common\tools.ps1
+          InitializeDotNetCli -install:$true
+          .\.dotnet\dotnet workload install aspire
+        displayName: Install .NET and Aspire Workload
 
-          # We'll need to give this service connection permission to get an auth token for PCS
-          - task: AzureCLI@2
-            inputs:
-              azureSubscription: $(serviceConnectionName)
-              scriptType: pscore
-              scriptLocation: inlineScript
-              inlineScript: |
-                $(Pipeline.Workspace)/ProductConstructionService.Deployment/ProductConstructionService.Deployment.exe `
-                  --subscriptionId $(subscriptionId) `
-                  --resourceGroupName $(resourceGroupName) `
-                  --containerRegistryName $(containerRegistryName) `
-                  --containerAppName $(containerappName) `
-                  --workspaceName $(containerappWorkspaceName) `
-                  --containerJobNames $(containerjobNames) `
-                  --newImageTag $(newDockerImageTag) `
-                  --imageName $(containerName) `
-                  --azCliPath "$(azCliPath)" `
-                  --isCi true `
-                  --entraAppId $(MaestroAppId) `
-                  --redisConnectionString $(redisConnectionString)
-            displayName: Deploy container app
+      - powershell: .\eng\common\build.ps1 -restore
```

Comment:
Wait, this creeped back in here

Reply:
again wasn't even caught by git, something weird happened